inc/admin/metabox/src/components/controls/RadioImage.js

```diff
@@ -0,0 +1,42 @@
+const { RadioControl } = wp.components;
+const {compose} = wp.compose;
+const {withDispatch, withSelect} = wp.data;
+
+let getOptions = function(props) {
```

Comment:
As components are defined as functions, using a react hooks based approach, the specific methods should be defined inside the component. 

As the method is not reassigned later so it should be defined at `const`.